### PR TITLE
EOS-16528: Converted ini to yaml file and integrated confstor in sspl…

### PR DESCRIPTION
**Pull request by shriya-deshmukh**
…_test code.

## Problem Statement
<pre>
  <code>
    Story Ref (if any):
    Converted sspl config INI to yaml file, and integrated conf_stor in sspl_test
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Please add Problem decription here...
  </code>
</pre>
## Solution
<pre>
  <code>
    Please add summary of the change,List any dependencies that are required for this change.
  </code>
</pre>
## Sanity testing on RPM done
<pre>
  <code>
    - [ ] Yes
    - [ ] No
  </code>
</pre>
## Unit/Manual Testing Description
<pre>
  <code>
    Please describe the tests that you ran to verify your changes.
  </code>
</pre>

### CODE REVIEW

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+   mgmt_interface: cliapi
+
+SYSTEM_INFORMATION:
+    cli_type: CS-A
+    product: LDR_R2
+    setup: cortx
+    data_path: /var/cortx/sspl/data/
+    sspl_log_file_path: /var/log/cortx/sspl/sspl.log
+    operating_system: centos7
+    log_level: INFO
```
Comment (mariyappanp) [inline low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml:219]: Need to keep common config in a separate file.

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+   threaded: true
+   probe: sysfs
+
+CPUFAULTSENSOR:
+   monitor: true
+   threaded: true
+   probe: sysfs
+
+STORAGE_ENCLOSURE:
+   mgmt_interface: cliapi
```
Comment (mariyappanp) [inline low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml:210]: Please ensure this is added in provisioner sheet where all the global common config present.

File: low-level/files/opt/seagate/sspl/setup/conf_based_sensors_enable [hunk 3]
```diff
@@ -73,14 +73,12 @@ if SSPL_STORE_TYPE == "consul":
         print("Error in connecting with consul: {}".format(cerror))
 
 # Update sensor information for sspl_test
-test_file_config_path=f"/opt/seagate/{PRODUCT_FAMILY}/sspl/sspl_test/conf/sspl_tests.conf"
-config = configparser.ConfigParser(allow_no_value=True)
-config.read(test_file_config_path)
+test_file_config_path=f"/opt/seagate/{PRODUCT_FAMILY}/sspl/sspl_test/conf/sspl_tests.yml"
+Conf.load('test', f'yaml://{test_file_config_path}')
```
Comment (mariyappanp) [inline low-level/files/opt/seagate/sspl/setup/conf_based_sensors_enable:77]: This may fail if any other component on this system already used this index.
Better keep the index as 'sspl_test'.

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml (hunk 1 above)
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml:219]: provisioner will not provide these values, hence i have added these in sspl config file.

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1.yml [hunk 4]
```diff
@@ -0,0 +1,199 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1.yml:1]: extension should be .yaml rt?

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml (hunk 2 above)
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml:210]: for this key provisinor commented that this is not supported by them. So I have kept it in sspl config file.

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+
+REALSTORFANSENSOR:
+   threaded: true
+
+REALSTORCONTROLLERSENSOR:
+   threaded: true
+
+REALSTORDISKSENSOR:
+   threaded: true
+   polling_frequency_override: 0
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml:158]: Can we have this polling_frequency_override for all realstor sensors mentioned here now. corresponding code to check polling_frequency_override exists in all realstor sensors.

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml [hunk 6]
```diff
@@ -0,0 +1,220 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml:1]: yaml extension?

File: low-level/files/opt/seagate/sspl/setup/conf_based_sensors_enable (hunk 3 above)
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/setup/conf_based_sensors_enable]: Done.

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml (hunk 2 above)
Comment (cdeshmukh) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml:210]: mgmt_interface need not be in global config, but in sspl.conf as its the interface SSPL uses for monitoring.

In LR R2 provisioning sheet, provisioner have already marked red for this one, stating they cant provide this.

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml (hunk 1 above)
Comment (cdeshmukh) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml:219]: product, setup, operating_system are global configs. should be referred from global configs ( temporary yaml file or provisioner supplied backend)

File: low-level/files/opt/seagate/sspl/conf/sspl_common_config.yml [hunk 7]
```diff
@@ -13,48 +13,39 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
 
-[SYSTEM_INFORMATION]
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/conf/sspl_common_config.yml:16]: Comment in https://github.com/Seagate/cortx-monitor/pull/281/files#r560360605 applies

Check comments for sspl.conf.LDR_R2 also in same PR

File: sspl_test/conf/sspl_tests.yml [hunk 8]
```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```
Comment (cdeshmukh) [inline sspl_test/conf/sspl_tests.yml:1]: same comments as other yaml file comments

File: sspl_test/constants.sh [hunk 9]
```diff
@@ -16,4 +16,7 @@
 PRODUCT_NAME='LDR_R2'
```
Comment (cdeshmukh) [inline sspl_test/constants.sh:16]: do we need this file even now, given all setup py scripts will be using sspl_constants.py

File: sspl_test/run_qa_test.sh [hunk 10]
```diff
@@ -112,7 +112,7 @@ deleteMockedInterface()
 
 kill_mock_server()
 {
-    # Kill mock API server
+    Kill mock API server
```
Comment (cdeshmukh) [inline sspl_test/run_qa_test.sh:115]: why removed. this will break

File: sspl_test/run_qa_test.sh [hunk 11]
```diff
@@ -194,6 +206,14 @@ execute_test()
     $sudo $script_dir/run_sspl-ll_tests.sh $plan
 }
 
+if [ "$SSPL_STORE_TYPE" == "confstor" ]
+then
+    # Read common key which are needed to fetch comfstor config.
```
Comment (cdeshmukh) [inline sspl_test/run_qa_test.sh:211]: typo - confstor

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1.yml [hunk 12]
```diff
@@ -0,0 +1,199 @@
...
+    syslog_host: localhost
+    syslog_port: 514
+
+RABBITMQINGRESSPROCESSOR:
+    virtual_host: SSPL
+    queue_name: actuator-req-queue
+    exchange_name: sspl-in
+    routing_key: actuator-req-key
+    username: sspluser
+    password: gAAAAABelEy2-hxLq8uThrKVZeb8ZY34cUPMYag8aMHJEHpXbaCie2cmeFsNx2nHBMCLLS2Bj9VUp9iSnyYDmywUHI7SsCoRIw==
```
Comment (thavanathan) [inline low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1.yml:39]: rabbitmq password will be different for different setups. Also not recommended to store the password in plain files.

File: low-level/files/opt/seagate/sspl/setup/conf_based_sensors_enable [hunk 13]
```diff
@@ -73,14 +73,12 @@ if SSPL_STORE_TYPE == "consul":
         print("Error in connecting with consul: {}".format(cerror))
 
 # Update sensor information for sspl_test
-test_file_config_path=f"/opt/seagate/{PRODUCT_FAMILY}/sspl/sspl_test/conf/sspl_tests.conf"
-config = configparser.ConfigParser(allow_no_value=True)
-config.read(test_file_config_path)
+test_file_config_path=f"/opt/seagate/{PRODUCT_FAMILY}/sspl/sspl_test/conf/sspl_tests.yml"
```
Comment (thavanathan) [inline low-level/files/opt/seagate/sspl/setup/conf_based_sensors_enable:76]: Please read this path from sspl_constants.py

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml (hunk 1 above)
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml]: "product, setup, operating_system" moved to sample_global_cortx_config.yaml file.

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1.yml (hunk 4 above)
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1.yml]: done.

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml (hunk 5 above)
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml]: Done, polling_frequency_override added for all real_stor sensors.

File: sspl_test/constants.sh (hunk 9 above)
Comment (shriya-deshmukh) [reply sspl_test/constants.sh:16]: yes, in run_qa_tests.sh  we are fetching data from this file.

File: sspl_test/run_qa_test.sh (hunk 10 above)
Comment (shriya-deshmukh) [reply sspl_test/run_qa_test.sh]: corrected.

File: sspl_test/run_qa_test.sh (hunk 11 above)
Comment (shriya-deshmukh) [reply sspl_test/run_qa_test.sh]: done

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1.yml (hunk 12 above)
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1.yml]: removed password from all files.

File: low-level/files/opt/seagate/sspl/setup/conf_based_sensors_enable (hunk 13 above)
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/setup/conf_based_sensors_enable]: done.

File: low-level/files/opt/seagate/sspl/conf/sspl_common_config.yml (hunk 7 above)
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/conf/sspl_common_config.yml]: addressed comments from  https://github.com/Seagate/cortx-monitor/pull/281 for  sspl.conf.LDR_R2.

File: low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy.sh [hunk 14]
```diff
@@ -272,9 +272,10 @@ fi
     # TODO: When RE brings main branch as same strucure as cortx_builds,
     # below should be removed and setup_yum_repo should be able to give cortx-py-utils.
     pkg_name="cortx-py-utils"
-    build_url="http://cortx-storage.colo.seagate.com/releases/cortx/github/main/centos-7.8.2003/cortx-utils_last_successful/"
+    build_url="http://cortx-storage.colo.seagate.com/releases/cortx/components/github/main/centos-7.8.2003/dev/cortx-utils/last_successful/"
+    yum install -y gcc rpm-build python36 python36-pip python36-devel python36-setuptools openssl-devel libffi-devel
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy.sh:276]: Lot of additional packages pulled here, most of them would be available on cortx build systems already. Why needed for deployment ?

File: sspl_test/constants.sh (hunk 9 above)
Comment (cdeshmukh) [reply sspl_test/constants.sh:16]: Ok

**Review by thavanathan — APPROVED** [review]

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1.yaml [hunk 15]
```diff
@@ -0,0 +1,199 @@
...
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+# Version 1.0.0
+
+SYSTEM_INFORMATION:
+    operating_system: centos7
```
[the remark this answers is not in the document]
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1.yaml]: added in global config.

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1.yaml [hunk 16]
```diff
@@ -0,0 +1,199 @@
...
+   transmit_interval: 10
+   units: MB
+   disk_usage_threshold: 80
+   cpu_usage_threshold: 80
+   host_memory_usage_threshold: 80
+
+NODEDATA:
+   probe: sysfs
+
+RARITANPDU:
```
[the remark this answers is not in the document]
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1.yaml]: done

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yaml [hunk 17]
```diff
@@ -0,0 +1,224 @@
...
+   user: admin
+   pass: admin
+
+BMC_INTERFACE:
+   default: system
+
+REALSTORSENSORS:
+   monitor: true
+   polling_frequency: 30
+   polling_frequency_override: 0
```
[the remark this answers is not in the document]
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yaml:142]: done

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yaml [hunk 18]
```diff
@@ -0,0 +1,224 @@
...
+   ack_routing_key: sensor-key
+   username: sspluser
+   password: 
+   message_signature_username: sspl-ll
+   message_signature_token: ALOIUD986798df69a8koDISLKJ282983
+   message_signature_expires: 3600
+   iem_route_addr: 
+   iem_route_exchange_name: sspl-out
+   primary_rabbitmq_host: localhost
+   limit_consul_memory: 50000000
```
[the remark this answers is not in the document]
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yaml:101]: added "TODO."

File: low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy.sh [hunk 19]
```diff
@@ -273,9 +273,9 @@ fi
     # below should be removed and setup_yum_repo should be able to give cortx-py-utils.
     pkg_name="cortx-py-utils"
     build_url="http://cortx-storage.colo.seagate.com/releases/cortx/components/github/main/centos-7.8.2003/dev/cortx-utils/last_successful/"
-    yum install -y gcc rpm-build python36 python36-pip python36-devel python36-setuptools openssl-devel libffi-devel
+    yum install -y gcc rpm-build python36-devel openssl-devel libffi-devel
     yum install -y $build_url/$(curl -s $build_url/|grep $pkg_name|sed 's/<\/*[^"]*"//g'|cut -d"\"" -f1) ||:
-    python3 -m pip install toml pyyaml idna==2.8 
+    python3 -m pip install toml pyyaml
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy.sh:278]: Need toml also? Where we are using toml?

File: low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy.sh (hunk 14 above)
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy.sh]: removed python36-setuptools, python36 python36-pip.

File: low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy.sh (hunk 19 above)
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy.sh:278]: I am not using it in my patch, but this seems cortx-py-utils pkg dependancy. 
while testing "sspl self provionser patch", we were getting error for this pkg, thats why mariyappan added this pkg.
@mariyappanp  can you please confirm.

I will test my patch without using these pkg on fresh vm.

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml (hunk 2 above)
Comment (mariyappanp) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml]: Okay.

**Review by mariyappanp — APPROVED** [review]

File: low-level/files/opt/seagate/sspl/conf/sample_global_cortx_config.yaml [hunk 20]
```diff
@@ -13,48 +13,40 @@
...
-monitor=true
-
-[RABBITMQCLUSTER]
-cluster_nodes=localhost
+cluster:
+   cluster_id: 'CortxCluster001'
+   product: LDR_R2
+   server_nodes:
+   srvnode_1:
+      operating_system: centos7
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/conf/sample_global_cortx_config.yaml:21]: OS needs to be read from /etc/system-release by sspl, so even if we keep in this sample file, Provisioner wont support this later and our code may break. So better remove this from here too.
This applies to any config which we are keeping here and Provisioner sheet has marked it red(means provisioner wont support).

File: low-level/files/opt/seagate/sspl/conf/sample_global_cortx_config.yaml [hunk 21]
```diff
@@ -13,48 +13,40 @@
...
-
-[NODEHWSENSOR]
-monitor=true
-
-[SYSTEMDWATCHDOG]
-monitor=true
-
-[RABBITMQCLUSTER]
-cluster_nodes=localhost
+cluster:
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/conf/sample_global_cortx_config.yaml:16]: Match the entries SSPL needs here with that in Provisioner sheet and KV pairs there. What exists here should reflect in that sheet and vice versa, atleast what we need.
like machine-id to nodename mapping we would need, as we are not supposed to use minion id now.

File: low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy.sh [hunk 22]
```diff
@@ -272,9 +272,9 @@ fi
     # TODO: When RE brings main branch as same strucure as cortx_builds,
     # below should be removed and setup_yum_repo should be able to give cortx-py-utils.
     pkg_name="cortx-py-utils"
-    build_url="http://cortx-storage.colo.seagate.com/releases/cortx/github/main/centos-7.8.2003/cortx-utils_last_successful/"
+    build_url="http://cortx-storage.colo.seagate.com/releases/cortx/components/github/main/centos-7.8.2003/dev/cortx-utils/last_successful/"
+    yum install -y gcc rpm-build python36-devel openssl-devel libffi-devel
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy.sh:276]: this line had my review comment earlier. We are not supposed to pull in new packages, or are these already being used in cortx, we are just deploying during dev deploy ? In which case, its fine.

File: sspl_test/framework/utils/load_conf_store.py [hunk 23]
```diff
@@ -0,0 +1,23 @@
...
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
```
Comment (cdeshmukh) [inline sspl_test/framework/utils/load_conf_store.py:17]: add description, what this script meant for

File: sspl_test/run_qa_test.sh [hunk 24]
```diff
@@ -23,12 +23,16 @@ IS_VIRTUAL=$(facter is_virtual)
...
-source $script_dir/constants.sh
-SSPL_STORE_TYPE=${SSPL_STORE_TYPE:-consul}
+source "$script_dir"/constants.sh
+SSPL_STORE_TYPE=confstor
 
 plan=${1:-}
 avoid_rmq=${2:-}
 
+common_config=yaml:///etc/cortx/sample_global_cortx_config.yaml
+test_config=yaml:///opt/seagate/"$PRODUCT_FAMILY"/sspl/sspl_test/conf/sspl_tests.yaml
```
Comment (cdeshmukh) [inline sspl_test/run_qa_test.sh:33]: sspl_tests.conf only we can keep, yaml being internal format(same as sspl.conf)

File: sspl_test/run_qa_test.sh [hunk 25]
```diff
@@ -194,6 +211,16 @@ execute_test()
     $sudo $script_dir/run_sspl-ll_tests.sh $plan
 }
 
+if [ "$SSPL_STORE_TYPE" == "confstor" ]
+then
+    # Read common key which are needed to fetch confstor config.
+    machine_id=`cat /etc/machine-id`
+    minion_id=`conf $common_config get "cluster>server_nodes>$machine_id"`
```
Comment (cdeshmukh) [inline sspl_test/run_qa_test.sh:218]: minion id is no more recommended to be used. but from machineid-to-nodename mapping, which will have value as srvnode-1, etc , same as we had in minion-id. @sandeepsgt @shriya-deshmukh   we clarified this earlier, please incorporate accordingly

File: sspl_test/rabbitmq_start_checker [hunk 26]
```diff
@@ -36,8 +38,8 @@ component = 'sspl/config'
 RABBITMQ_CLUSTER_SECTION = 'RABBITMQCLUSTER'
 host = os.getenv('CONSUL_HOST', CONSUL_HOST)
 port = os.getenv('CONSUL_PORT', CONSUL_PORT)
-SSPL_STORE_TYPE = 'file'
-sspl_config_file = '/etc/sspl.conf'
+SSPL_STORE_TYPE = 'confstor'
+sspl_config_file = '/etc/cortx/sspl.conf'
```
Comment (cdeshmukh) [inline sspl_test/rabbitmq_start_checker:42]: why this change?

File: low-level/files/opt/seagate/sspl/conf/sample_global_cortx_config.yaml (hunk 21 above)
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/conf/sample_global_cortx_config.yaml:16]: Updated file according to provisinor sheet. (added setup and product "release>setup, release>product")

File: low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy.sh (hunk 22 above)
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy.sh:276]: Conf store cli will not work without these pkgs,

File: low-level/files/opt/seagate/sspl/conf/sample_global_cortx_config.yaml (hunk 20 above)
Comment (shriya-deshmukh) [reply low-level/files/opt/seagate/sspl/conf/sample_global_cortx_config.yaml]: removed.

File: sspl_test/framework/utils/load_conf_store.py (hunk 23 above)
Comment (shriya-deshmukh) [reply sspl_test/framework/utils/load_conf_store.py]: purpose of this script is to load conf stor. but #281  PR has related changes for this,  this is not required anymore, removing this file.

File: sspl_test/run_qa_test.sh (hunk 24 above)
Comment (shriya-deshmukh) [reply sspl_test/run_qa_test.sh]: done

File: sspl_test/run_qa_test.sh (hunk 25 above)
Comment (shriya-deshmukh) [reply sspl_test/run_qa_test.sh]: Internally I am using machine id only to map the node name, just used minion-id as variable name  , 
renamed it to srvnode in latest commit.

File: sspl_test/rabbitmq_start_checker (hunk 26 above)
Comment (shriya-deshmukh) [reply sspl_test/rabbitmq_start_checker]: Removed this change.
